sonatype-ci-release/src/main/scala/org/typelevel/sbt/TypelevelSonatypeCiReleasePlugin.scala

```diff
@@ -70,22 +69,17 @@ object TypelevelSonatypeCiReleasePlugin extends AutoPlugin {
 
   private def tlCiReleaseCommand: Command =
     Command.command("tlCiRelease") { state =>
-      val newState = Command.process("tlRelease", state)
+      val newState = Command.process("tlRelease", state, _ => ())
       newState.getSetting(version).foreach { v =>
-        val resolver = newState.getSetting(sonatypeDefaultResolver).fold("") {
-          case repo: MavenRepository =>
+        val resolver = newState.getSetting(isSnapshot).fold("") {
+          case true =>
             s"""|```scala
-                |resolvers += "${repo.name}" at "${repo.root}"
+                |resolvers += "central-snapshots" at "https://central.sonatype.com/repository/maven-snapshots/"
                 |```
                 |""".stripMargin
 
-          case repo: URLRepository =>
-            s"""|```scala
```

Comment:
This is untested.  I think the point is to get a snippet into the Actions job summary?  I'm not even sure if they're retrievable from that URL.